fetch-node.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
 var fetch = require('node-fetch');
+var URL = require('url').URL;
 
 function wrapFetchForNode(fetch) {
   // Support schemaless URIs on the server for parity with the browser.
   // https://github.com/matthew-andrews/isomorphic-fetch/pull/10
   return function (u, options) {
+    if (u instanceof URL) {
```

Comment:
I like what you're doing here, but it may be better to treat the string as a special case and forward everything else, i.e.

```javascript
return function(u, options) {
  if (typeof u === 'string' && u.slice(0, 2) === '//') {
    return fetch('https:' + u, options);
  }

  return fetch(u, options);
};
```

Reply:
I agree.